client/mysqlbinlog.cc

```diff
@@ -3169,7 +3169,7 @@ int main(int argc, char** argv)
   /* Set delimiter back to semicolon */
   if (retval != ERROR_STOP)
   {
-    if (!stop_event_string.is_empty())
+    if (!stop_event_string.is_empty() && result_file)
```

Comment:
@an3l @LinuxJedi I am confused about this check only happening here.

If this check is necessary here, why is the fprintf at line 3161 legal? That one doesn't check for opt_raw_mode either. Is `opt_raw_mode` mutually exclusive with `opt_flashback`?

Reply:
@cvicentiu  coverity [report](https://scan5.scan.coverity.com/reports.htm#v58936/p10357/fileInstanceId=231728438&defectInstanceId=53074517&mergedDefectId=1519690&eventId=53074517-21) said that in case when`opt_raw_mode` is used and `opt_flasback` is 0, `result_file` is assigned to 0 (line 3071) and when comes to this line, `fprintf` is used on 0.